test/e2e/test_cluster_configs/windows/vhd_url.json

```diff
@@ -0,0 +1,52 @@
+{
+	"env": {
+		"REGION_OPTIONS": "westus",
+		"WINDOWS_NODE_VHD_URL": "https://aksenginee2etestimages.blob.core.windows.net/vhds/WS2019-1908.vhd"
+	},
+	"apiModel": {
+		"apiVersion": "vlabs",
+		"properties": {
+			"orchestratorProfile": {
+				"orchestratorType": "Kubernetes",
+			},
+			"masterProfile": {
+				"count": 1,
+				"dnsPrefix": "",
+				"vmSize": "Standard_D2_v2"
+			},
+			"agentPoolProfiles": [
+				{
```

Comment:
1 linux agent and 2 windows agents should be sufficient for /windows/vhd_url.json, /windows/shared_image_gallery.json, and /windows/image_reference.json